Makefile

```diff
@@ -262,26 +262,26 @@ jstest: build-addons build-addons-napi ## Runs addon tests and JS tests
 .PHONY: test
 # This does not run tests of third-party libraries inside deps.
 test: all ## Runs default tests, linters, and builds docs.
-	# Build the addons before running the tests so the test results
```

Comment:
Is there any reason for the white space changes?

Reply:
With the indentation the comments are sent to the shell and displayed as:
```shell
# Build the addons before running the tests so the test results
```
My reasoning was that this was a mistake and that if we wanted the comments to be displayed the echo command would be used instead and without the `#`.

Reply:
I agree with @danbev .  I'll also note that the addons are built twice and the linters aren't run.  See https://github.com/nodejs/node/pull/22031 (I got distracted by work and now am on vacation).  If this pull request could clean some of this up, that would be great.  Otherwise, I'll take a look at cleaning up the rest when I get back.